data/pizza.json

```diff
@@ -975,8 +975,8 @@
 				"hidden_ingredients": [
 					"tomatsås", "ost"
 				],
-				"price": 110,
-				"family_price": 240
+				"price": 115,
+				"family_price": 245
 			},
 			"ruccola": {
```

Comment:
[Luftorkad Skinka instead of parmaskinka](https://sannegarden.com/meny/johanneberg/#:~:text=Oliver%2C%20Soltorkade%20Tomater%2C%20Mozzarellaost%2C%20Salami%2C%20Luftorkad%20Skinka%2C%20Ruccola%20Sallad%2C%20Vitl%C3%B6k)